docs/public/specifications/test-zones/Zone-TP/zone11.md

```diff
@@ -90,7 +90,7 @@ irrelevant to the test scenario and must be ignored.
 * (1) All tags except for those specified as "Forbidden message tags" (no instances for these test scenarios)
 * (2) All tags except for those specified as "Mandatory message tags"
 
-## Test scenarios and setup of test zones
+## Zone setup for test scenarios

```

Comment:
The section actually defines the scenario features, so the existing header is probably better. Also used in many test scenario specifications.